teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < person
+  def initialize(specialization, _parent_permission, name = 'Unknown', age: true)
+    super(age, parent_permission, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Kindly note that The parent_permission should have the default value of `true`. Hint :point_down: 

```rb
 def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

Reply:
Thank you for your comment! Should be corrected.